### ISSUE TITLE
Sniff to forbid useless {@inheritdoc}

### ISSUE THREAD

**Issue by Majkl578** [slevomat/coding-standard#399]
There is a lot of code using:
```php
/**
 * {@inheritDoc}
 */
```
Quoting [phpDoc standard](https://docs.phpdoc.org/guides/inheritance.html#the-inheritdoc-tag):
> Important: Currently some applications have DocBlocks containing just the `{@inheritDoc}` inline tag to indicate that their complete contents should be inherited. This usage breaks with the PHPDoc Standard as summaries cannot contain inline tags and **inheritance is automatic; you do not need to define a special tag for it**.

In other words, doc block containing only `{@inheritDoc}` is pointless and should be reported & removed as useless.

**Comment by kukulich** [comment]
Implemented in https://github.com/slevomat/coding-standard/commit/fc3dc45dec64d511706acd4f7a092e311e245eae
